README.md

```diff
@@ -77,7 +81,62 @@ variables to your root queries and containers:
   component={Widgets}
   queries={WidgetsQueries}
   queryParams={['date', 'color']} // date and color will be passed as variables
-/>`
+/>
+```
+
+If you’re passing on params from path components, like a URL containing slugs, make sure to prefix your params with a colon:
```

Comment:
Params from path components are actually always used (and we use them from the active path as well as all of its ancestors).